SnapWar/ViewControllers/BattleViewController.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+class BattleViewController: UIViewController {
+    let fullScreenImageView = SWFullscreenImageView(named: "test-image-2")
```

Comment:
If you try running the app with this image, it looks messed up until you do an orientation change at least once. The other images are fine though if you test those (`test-image-1`, `test-image-3`, `test-image-4`)

Reply:
hmmm, what's messed up about the images?

Reply:
looks like this
<img width="520" alt="Screenshot 2020-04-12 at 23 57 16" src="https://user-images.githubusercontent.com/14267246/79080965-6e83b080-7d19-11ea-8fe2-f81dda1db3d7.png">

Reply:
hmmmm, let me look into what's causing it